test/test-tcp-oob.c

```diff
@@ -98,6 +98,11 @@ static void connection_cb(uv_stream_t* handle, int status) {
 
 
 TEST_IMPL(tcp_oob) {
+
+#if defined(__MVS__)
+  RETURN_SKIP("Not supported by the platform");
```

Comment:
maybe its obvious, but the Z TCP stack doesn't implement OOB messages? Maybe the not supported message should say what isn't supported, its a bit more clear than having to read the test to see exactly what it is.